source/index.txt

```diff
@@ -1,35 +1,111 @@
-   :depth: 2
-   :class: singlecol
+.. introduction::
 
-The {+atlas-cli+} is a command line interface built specifically for |service|. 
-After you install and configure the {+atlas-cli+}, you can manage |service|
-{+database-deployments+} from the terminal with:
+   The {+atlas-cli+} is a command line interface built specifically for 
+   |service-fullname|. It lets you interact with your |service| 
+   {+database-deployments+} and Atlas Search from the terminal with
+   short, intuitive commands, so you can accomplish complex database management tasks in seconds.
 
-- Short, intuitive commands. While you can use ``curl`` commands to access the
-  |service| APIs from the terminal, {+atlas-cli+} commands are much shorter.
-- Fewer system resources. Managing |service| from the terminal requires less
-  available random access memory (RAM) and CPU processing time than the |service| UI.
+   .. button:: Install the {+atlas-cli+}
+      :uri: https://www.mongodb.com/docs/atlas/cli/stable/install-atlas-cli/
 
-.. include:: /includes/fact-use-mongocli-for-om-cm.rst
+   Then, run ``atlas setup``
 
-Follow these steps to get started using the {+atlas-cli+}:
+.. image:: /images/hero.png
+   :alt: Homepage hero image
 
-1. :ref:`Install the {+atlas-cli+} <install-atlas-cli>`
-2. :ref:`Connect from the {+atlas-cli+} <connect-atlas-cli>`
-3. :ref:`See tutorials to manage Atlas from the {+atlas-cli+}
-   <atlas-cli-tutorials>` or 
-   :doc:`Go straight to the {+atlas-cli+} commands </command/atlas>`
+.. kicker:: What You Can Do
+
+Manage Atlas from the Command Line
+----------------------------------
+
+Get started quickly with two commands:
+
+.. tabs::
+
+   .. tab:: Install the {+atlas-cli+}
+      :tabid: install
+
+      .. procedure::
+
+         .. step:: Run ``brew install mongodb-atlas-cli``
+            
+            See our :ref:`documentation <install-atlas-cli>` for more
+            ways to install the {+atlas-cli+}.
+
+      .. image:: /images/install.gif
+         :alt: brew install mongodb-atlas-cli command
+
+   .. tab:: Set Up Atlas
+      :tabid: setup
+
+      .. procedure::
+
+         .. step:: Run ``atlas setup``
+
+            ``atlas setup`` does all of the following with one command:
+
+            1. Signs up for an |service| account.
+            #. Authenticates with the new |service| account.
+            #. Creates one free database.
+            #. Loads :atlas:`sample data 
+               </sample-data/available-sample-datasets/>` 
+               into your |service| database.
+            #. Adds your IP address to your project's IP access list.
+            #. Creates a MongoDB user for your |service| 
+               {+database-deployment+}.
+            #. Connect to your new {+database-deployment+} using the
+               MongoDB Shell, {+mongosh+}.
+
+            To learn more, see the :doc:`atlas setup documentation </command/atlas-setup>`.
+
+      .. image:: /images/setup.gif
+         :alt: atlas setup command
```

Comment:
Thought: I don't really feel like we're using this section right... its more for use cases. In my mind, the first tab would be "Get Started with Atlas CLI" and would include step 1: Install then step 2: setup. And the next tab would probably be create a cluster or some other common task we want to highlight.

I know we are down to the wire here now so I'm open to discussing after MDBW.

Reply:
This section looked different when I first put together the PLP document for design, but Jakub requested this content on the tabs. Additionally, we had plans for a more in-depth tutorial covering several topics to go on a 3rd tab but we won't have time to do that before MDBW. I think we should get this PR merged for now and I will add revisiting this section to a ticket to talk about some overall changes.